src/mon/PGMap.cc

```diff
@@ -3238,6 +3238,14 @@ void PGMap::get_health_checks(
 	summary += " reporting legacy (not per-pool) BlueStore omap usage stats";
       } else if (asum.first == "BLUESTORE_SPURIOUS_READ_ERRORS") {
         summary += " have spurious read errors";
+      } else if (asum.first == "BLUESTORE_SLOW_OP_ALERT") {
+        summary += " experiencing slow operations in BlueStore";
+      } else if (asum.first == "BLOCK_DEVICE_STALLED_READ_ALERT") {
```

Comment:
and even more - do we really need device type in the alert keyword?
Wouldn't be BDEV_STALLED_READ_ALERT sufficient for all the alerts from Block Device? We mention the device type in the alert message anyway.

Reply:
This is a map `std::map<std::string,std::string> osd_alert_list_t`. If we don't keep different key, eventually message will be replaced.

Reply:
Should I remove the summary?

Reply:
> This is a map `std::map<std::string,std::string> osd_alert_list_t`. If we don't keep different key, eventually message will be replaced.

ah ok, makes sense to have different keywords then

Reply:
> Should I remove the summary?

I'd really like to see the actual output from 'ceph health detail' with this alert raised. Can you simulate one?
My feeling is that the overall summary would contain duplicate information...